listenbrainz/webserver/views/user_timeline_event_api.py

```diff
@@ -98,6 +99,56 @@ def create_user_recording_recommendation_event(user_name):
     return jsonify(event_data)
 
 
+@user_timeline_event_api_bp.route('/user/<user_name>/timeline-event/create/notification', methods=['POST', 'OPTIONS'])
+@crossdomain(headers="Authorization, Content-Type")
+@ratelimit()
+def create_user_notification_event(user_name):
+    """ Post a message with a link on a user's timeline. Only approved users are allowed to perform this action.
+
+    The request should contain the following data:
+
+        {
```

Comment:
how will this format in the documentation? It might need a prefix to tell it to highlight it as json

Reply:
I think it should show correctly as [this](https://github.com/metabrainz/listenbrainz-server/blob/3cf1d79400123bcb2f3cf111c7941bd9957f23a7/listenbrainz/webserver/views/api.py#L385) does [here](https://listenbrainz.readthedocs.io/en/production/dev/api/#get--1-validate-token)